### ISSUE TITLE
[FEATURE] Version Page in menu

### ISSUE THREAD

**Issue by phulstaert** [Leantime/leantime#882]
I was looking for the version of Leantime I was running and remembered this was displayed in the footer somewhere. 
I think (but not entirely sure) this disappeared with the new theme.

I think it would be a useful feature if users with admin roles had an extra menu called "Version" or "Information" or something in the lines of that.
In this screen, we could display:

- The current Leantime version (and if there is an update available). 
If there is an update available, this should be a link to the release page of GitHub until we have an easy way to start an update using the webinterface.
- The current database version
- The size of the database
```
SELECT ROUND(sum(((data_length + index_length) / 1024 / 1024)), 2) AS "Size (MB)" FROM information_schema.TABLES
WHERE table_schema = "' . $_ENV['LEAN_DB_DATABASE'] . '";
```
- Amount of free space (the public part where we store our logo/avatars/...)
```
echo round( disk_free_space($_SERVER['DOCUMENT_ROOT'] . "/userfiles") / 1024 / 1024, 2) . " MB";
//disk_free_space is in bytes
```
- MySQL version
```
$conn = new PDO( 'odbc:sample', 'db2inst1', 'ibmdb2' );
echo $conn->getAttribute( constant( "PDO::ATTR_SERVER_VERSION" ));
```
- PHP version
```
echo phpversion();
or
echo $_SERVER['PHP_VERSION'];
```
- Server version
```
echo $_SERVER['SERVER_SOFTWARE'];
```

Add more if you can think of it.

**Comment by ashelman-ec** [comment]
You are correct that in older versions it does display in the footer. I am running v2.1.9 and here is a screenshot.

I agree that this information is necessary, both in order to know whether one has the [latest release](https://github.com/Leantime/leantime/releases), and for reporting bugs per the bug template (as are the PHP and MySQL versions).

![Version](https://user-images.githubusercontent.com/104984057/186668549-5342eb7d-2568-42a7-b857-4a39f83c4ef7.png)

**Comment by phulstaert** [comment]
Hi @ashelman-ec , 

I can see it again. I don't know it if was gone in a certain version, but it is back. That is the most important thing :-)

![image](https://user-images.githubusercontent.com/8971696/186669778-66c297fd-4f13-4463-bcd9-d0863cfe1894.png)

This does not remove the fact that the page in my feature-request is unneeded. This provides a lot of usefull information.

**Comment by marcelfolaron** [comment]
Its back as of 2.2.5
let me know if you dont see it there
